tests/testthat/test-methods.R

```diff
@@ -107,36 +101,35 @@ test_that("print.lint works for inline data, even in RStudio", {
 
   l <- lint("x = 1\n")
 
-  # Make sure lints print to console.
-  # The full output is:
-  #
-  # > <text>:1:3: style: Use <-, not =, for assignment.
-  # > x = 1
-  # >   ^
```

Comment:
This is exactly what snapshot testing is for. We are rarely going to check this output manually.